tests/test_views_records.py

```diff
@@ -103,7 +105,9 @@ def test_records_are_isolated_by_bucket_and_by_collection(self):
                           MINIMALIST_COLLECTION,
                           headers=self.headers)
         other = self.record_url.replace('barley', 'ba').replace('beers', 'be')
-        self.app.get(other, headers=self.headers, status=404)
+        response = self.app.get(other, headers=self.headers, status=404)
+        self.assertEqual(response.json['details']['id'], self.record['id'])
+        self.assertEqual(response.json['details']['resource_name'], 'record')
```

Comment:
Here we don't need the assertion on the `details` fields. It's unrelated to the *specs* of this test:  «test_unknown_collection_does_not_query_timestamp»